### ISSUE TITLE
Make vanilla a mod

### ISSUE THREAD

**Issue by selaux** [ja2-stracciatella/ja2-stracciatella#434]
What do you think about making vanilla a mod and using some different `game.json` as a default that enables non-gameplay related changes (e.g. shortcuts, gui_extras, ...)

**Comment by JAsmine-ja2** [comment]
Very good idea to add "bits of chocolate" and still keep lots of vanilla, if I understand it right.

If there is a "vanilla" mod, than there is also a "stracciatella" mod, right?

**Comment by selaux** [comment]
Yes, though it will be the default and thus would not be explicitly enabled (it would basically be what is in `externalized` dir).

**Comment by lynxlynxlynx** [comment]
Sounds ok ... eventually. I don't think we have any controversial things yet, much less enabled by default. Window jumping will be one, but since it lacks animations, I wouldn't enable it by default even in a modded mode.
